### DISCUSSION TITLE
Error:Cannot download 'https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css': Remote host terminated the handshake

### DISCUSSION THREAD

**Discussion by mina20088** [twbs/bootstrap#38782] [Q&A] [answered]
### Discussed in https://github.com/orgs/twbs/discussions/38732

<div type='discussions-op-text'>

<sup>Originally posted by **mina20088** June  9, 2023</sup>
hi i have a problem when try to download the bootstrap from php storm when i right clicked on the link to download the lib i got this error 
![Screenshot 2023-06-08 224408](https://github.com/twbs/bootstrap/assets/39167278/66affb7f-deee-4b5f-8be4-4e020c19697e)
</div>

according the the issue number  #38764 i tried to take the cdn from the bootstrap website and i still have the same issue here there is something wrong with the cdn provider please check with the provider i cant access or download the lib through cdn 
note : please do not close the issue and say that there are an integrity problem because the problem is from the provider

**Comment by jimaek** [comment]
Please open the file in a browser and show the HTTP headers and contents you get

**Reply by batalelo** [reply]
All developers in egypt found that cdn.jsdelivr.net is blocked and not working from Egypt

Our ISP Provider is Blocked it, and we don't now why !!

 

But i found this and work perfectly, may it help anyone

cdn.jsdelivr.xyz

 

LIKE:

// load any project hosted on npm

https://cdn.jsdelivr.xyz/npm/package@version/file

 

// load any project hosted on npm

https://cdn.jsdelivr.xyz/npm/package@version/file


// load jQuery v3.6.4

https://cdn.jsdelivr.xyz/npm/jquery@3.6.4/dist/jquery.min.js


// use a version range instead of a specific version

https://cdn.jsdelivr.xyz/npm/jquery@3.6/dist/jquery.min.js

https://cdn.jsdelivr.xyz/npm/jquery@3/dist/jquery.min.js


// omit the version completely to get the latest one

// you should NOT use this in production

https://cdn.jsdelivr.xyz/npm/jquery/dist/jquery.min.js


// add ".min" to any JS/CSS file to get a minified version

// if one doesn't exist, we'll generate it for you

https://cdn.jsdelivr.xyz/npm/jquery@3.6.4/src/core.min.js


// omit the file path to get the default file

https://cdn.jsdelivr.xyz/npm/jquery@3.6


// add / at the end to get a directory listing

https://cdn.jsdelivr/.xyz/npm/jquery/

**Comment by mina20088** [comment]
@jimaek  you mean the html file that i am using the cdn

**Comment by mina20088** [comment]
@jimaek  this is the inspection -> console tap from browser 

![Screenshot from 2023-06-17 21-19-21](https://github.com/twbs/bootstrap/assets/39167278/073a85d6-f0e2-44d8-a982-f5624e51ce15)

**Comment by jimaek** [comment]
I mean show this https://dl.dropboxusercontent.com/s/w4cebanxhrb2ti0/chrome_2023-06-17_20-21-31.png

**Comment by mina20088** [comment]
@jimaek  here it is 
![Screenshot from 2023-06-17 21-23-52](https://github.com/twbs/bootstrap/assets/39167278/5f764e12-50e7-4b9c-9a85-f33be7f755dd)

**Comment by jimaek** [comment]
Including the HTTP headers and IP address please

**Comment by mina20088** [comment]
@jimaek 


here but Firefox dose not  show ip address
note : i am using Firefox

![Screenshot from 2023-06-17 21-36-33](https://github.com/twbs/bootstrap/assets/39167278/247f8bd5-ab62-4f4d-b99d-4423a956e52b)

**Comment by mina20088** [comment]
@jimaek  if this whould not help i will install chrome on my linux

**Comment by jimaek** [comment]
just ping cdn.jsdelivr.net using the command line on Windows

**Comment by mina20088** [comment]
@jimaek here is the result of the ping
![Screenshot from 2023-06-17 21-43-12](https://github.com/twbs/bootstrap/assets/39167278/d7286b16-7f07-4e64-a1f3-244a17af3e08)

**Comment by jimaek** [comment]
Are you based in Egypt?

**Comment by mina20088** [comment]
@jimaek yes

**Comment by jimaek** [comment, marked as answer]
Its not an issue on the jsDelivr side, some Egyptian ISPs are blocking the cdn.jsdelivr.net domain for some reason.
Try to contact your ISP provider to ask them to unblock it.
Additionally you can use a VPN or switch the hostname to fastly.jsdelivr.net

**Reply by Philo96** [reply]
To be honest, you made a favor for me by this answer ;)

**Comment by batalelo** [comment]
All developers in egypt found that cdn.jsdelivr.net is blocked and not working from Egypt

Our ISP Provider is Blocked it, and we don't now why !!

 

But i found this and work perfectly, may it help anyone

cdn.jsdelivr.xyz

 

LIKE:

// load any project hosted on npm

https://cdn.jsdelivr.xyz/npm/package@version/file

 

// load any project hosted on npm

https://cdn.jsdelivr.xyz/npm/package@version/file


// load jQuery v3.6.4

https://cdn.jsdelivr.xyz/npm/jquery@3.6.4/dist/jquery.min.js


// use a version range instead of a specific version

https://cdn.jsdelivr.xyz/npm/jquery@3.6/dist/jquery.min.js

https://cdn.jsdelivr.xyz/npm/jquery@3/dist/jquery.min.js


// omit the version completely to get the latest one

// you should NOT use this in production

https://cdn.jsdelivr.xyz/npm/jquery/dist/jquery.min.js


// add ".min" to any JS/CSS file to get a minified version

// if one doesn't exist, we'll generate it for you

https://cdn.jsdelivr.xyz/npm/jquery@3.6.4/src/core.min.js


// omit the file path to get the default file

https://cdn.jsdelivr.xyz/npm/jquery@3.6


// add / at the end to get a directory listing

https://cdn.jsdelivr/.xyz/npm/jquery/
